doc/apps/voila.md

```diff
@@ -1,17 +1,26 @@
 # Voilà
 
-```{tip}
-If you haven't deployed your first application, check out the [quickstart guide.](../quickstart/app.md)
+## Deploy an example
```

Comment:
let's try using tabs: one for UI (default) and another one for CLI

https://jupyterbook.org/en/stable/content/components.html#tab-content

it might look better

Reply:
Now using [tabs.](https://github.com/ploomber/doc/pull/138/files#diff-0bfee22e5235956a736cedde5812194226d112687d509d06c9295aa7679b7bb7R27-R43)